src/egon/data/datasets/power_etrago/match_ocgt.py

```diff
@@ -54,7 +56,13 @@ def insert_open_cycle_gas_turbines(scn_name="eGon2035"):
     # read carrier information from scnario parameter data
     scn_params = get_sector_parameters("gas", scn_name)
     gdf["efficiency"] = scn_params["efficiency"][carrier]
-    gdf["marginal_cost"] = scn_params["marginal_cost"][carrier]
+    gdf["marginal_cost"] = (
+        scn_params["marginal_cost"][carrier]
+        / scn_params["efficiency"][carrier]
+    )
+
+    # Adjust p_nom
+    gdf["p_nom"] = gdf["p_nom"] / scn_params["efficiency"][carrier]
```

Comment:
Amelia, could you confirm if "p_nom" should be divided or multiplied by the efficiency?

Reply:
p_nom should be divided by the efficiency because it should be bigger than the power at the electrical bus (see #863).

Reply:
Great.